modular_skyrat/modules/awaymissions_skyrat/mission_code/mothership_astrum.dm

```diff
@@ -178,9 +175,14 @@
 	mob_species = /datum/species/human
 	outfit = /datum/outfit/lobotomite
 	short_desc = "You are a lobotomite, an experiment of those who took you away from... why can't you remember?"
-	flavour_text = "Your head hurts, something feels different, you want to go home. \
-	Thinking about home, you can't remember where that is, or if it even existed. \
-	Your goal is to find out who you are, and escape from this place, work with anyone friendly you can find."
+	flavour_text = "You are the Lobotomite. \
+	Do not explore the gateway. \
+	Do not touch anything in the gateway. \
+	Do not use anything in the gateway.\
+	This is not meant to be a protagonist or an antagonist.\
+	You are meant to work with the exploration team.\
+	Do not attack anything in the gateway except in self-defense.\
+	DO NOT TOUCH ANYTHING IN THE GATEWAY."
```

Comment:
Ah yes, gut insightful flavor text in favor of "do not be a shitter". Also, you DO realize that it basically leaves them with nothing to do if explorers aren't present?

Reply:
I've literally given them a full room full of botanical tools, and maybe people should start thinking twice before spawning as a ghost role with no explorers. Besides, harping on me for removing flavour text in order to add "don't be a shitter" when people would literally just be nothing but shitters was a pain in the ass.

Reply:
It's more of "don't let them be shitters". Which is somewhat already done since, from what I can see anyway, they can't leave their room if no one lets them out regardless.

Reply:
What matters is their roleplay with the expeditionary force, which I find is more often than not super shitty. Besides, without that they'll somehow grow some botany bullshit to escape.

Reply:
You have:
Cherry bombs which are unobtainable.
Bluespace tomatoes and bananas, which I'm pretty sure can be countered with making the gateway area have the NO_TELEPORT (or whatever it's called) flag.

I honestly can't think of anything else you could use to escape.

Reply:
Some dumbass reagent bomb? Either way, we need rules in place to prevent people from shitting around, it's what Gandalf's already said that he wants for that text.

Reply:
...is the blast door NOT already indestructible?

Reply:
Also, pretty sure there's already a "important info" var that TG uses for ghost role rules. You could use that.

Reply:
If you can show me it, I'll use it.

Reply:
important_info
Used by hotel staff spawners to show the player not to leave the hotel.
Whatever string you set it as will be shown in big red letters in the spawners menu.